nodepool/nodepool.yaml

```diff
@@ -182,6 +189,35 @@ providers:
         private-key: /home/nodepool/.ssh/id_rsa
         name-filter: 'Performance'
         config-drive: true
+  - name: devstack-rocket-man-asr-prll
+    username: admin
+    password: secret
+    project-name: demo
+    auth-url: http://192.133.156.19/identity
+    api-timeout: 60
+    boot-timeout: 120
+    max-servers: 3
+    rate: 1
+    # Remove this when glean has support for ipv6 on centos
+    ipv6-preferred: false
+    image-type: qcow2
+    template-hostname: '{image.name}-{timestamp}'
+    images:
+      - name: ubuntu-xenial
+        min-ram: 8192
+        diskimage: ubuntu-xenial
+        username: jenkins
+        private-key: /home/nodepool/.ssh/id_rsa
+        name-filter: 'Performance'
+        config-drive: true
```

Comment:
Might as well remove this as we have no plan to use ubuntu for these jobs, and it'll just cause more issues with the number of images in glance.

Reply:
sure